webserver/views/user.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 import webserver
 import db.user
+import ujson
```

Comment:
Can you explain the problem faced with using ujson? Or can you point to the ticket?  It might be useful in future. :)